src/config.ts

```diff
@@ -239,9 +239,9 @@ export const MAX_DATA_ITEM_QUEUE_SIZE = +env.varOrDefault(
   '100000',
 );
 
-// The maximum number of bundles to queue for unbundling before skipping
-export const BUNDLE_DATA_IMPORTER_QUEUE_SIZE = +env.varOrDefault(
-  'BUNDLE_DATA_IMPORTER_QUEUE_SIZE',
+// The maximum number of data imports to queue for unbundling/importing before skipping
+export const DATA_IMPORTER_QUEUE_SIZE = +env.varOrDefault(
+  'DATA_IMPORTER_QUEUE_SIZE',
```

Comment:
This data import will still be for bundles so the name shouldn't change. This may require a change to pass this value into the data importer if it's referenced directly in the data downloader.

Reply:
it's used in the class I think for the queue size of the worker jobs of the class, for both cases, but I did revert this still

Reply:
> it's used in the class I think for the queue size of the worker jobs of the class, for both cases, but I did revert this still

Gotcha. I'll take a look when I rereview in a bit. We may want two constants now.